XRTK-Core/Packages/com.xrtk.core/Services/InputSimulationSystem/MixedRealityInputSimulationSystem.cs

```diff
@@ -0,0 +1,308 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using XRTK.Definitions.Devices;
+using XRTK.Definitions.InputSimulationSystem;
+using XRTK.Definitions.Utilities;
+using XRTK.Interfaces.InputSimulationSystem;
+using XRTK.Interfaces.InputSystem;
+using XRTK.Interfaces.Providers.Controllers;
+using XRTK.Providers.Controllers;
+using XRTK.Utilities;
+
+namespace XRTK.Services.InputSimulationSystem
```

Comment:
Can we change the namespace to 
```
XRTK.Services.InputSystem.Simulation
```
Makes more sense that way, as the input system is required for simulation, it's not a replacement